customer_satisfaction/README.md

```diff
@@ -0,0 +1,92 @@
+# End to End Customer Satisfaction Pipeline with ZenML 
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/zenml)](https://pypi.org/project/zenml/)
+
+**Problem statement**:- For a given historical data of the customer predict the review score for the next order or purchase. I will be using [Brazilian E-Commerce Public Dataset by Olist](https://www.kaggle.com/datasets/olistbr/brazilian-ecommerce) dataset, The data-set has information of 100k orders from 2016 to 2018 made at multiple marketplaces in Brazil. Its features allow viewing orders from multiple dimensions: from order status, price, payment, and freight performance to customer location, product attributes and finally reviews written by customers. The objective here is to predict the customer satisfaction score for a given order based on the given features like order status, price, payment, etc. I will be using the [ZenML](https://zenml.io/) framework to build a production ready pipeline to predict the customer satisfaction score for the next order or purchase.
+
+The Purpose of this repository is to demonstrate how [ZenML](https://github.com/zenml-io/zenml) empowers your business to build and deploy machine learning pipelines in a multitude of ways:
```

Comment:
We might want to be careful about how much we claim that this is a 'real business' use case. What we have in this ZenFile is a much smaller example. A real business example would be on a different scale.

Reply:
Agree with this but don't you think it will attract peoples as well